### ISSUE TITLE
Tutorial for creating bindings

### ISSUE THREAD

**Issue by WillCalderwood** [robovm/robovm-robopods#42]
Hi, not really an issue as such, but would you be able to create a tutorial for how to create bindings. Preferably aimed at those with zero iOS development experience. It would be useful in helping getting others involved rather than just having one person manning the ship.

I need to create an AppsFlyer binding but I have no idea where to start.

Many thanks

**Comment by keskin** [comment]
+1

**Comment by ASMCOM** [comment]
+1

**Comment by xeoshow** [comment]
+1

**Comment by obigu** [comment]
+1

**Comment by tajny** [comment]
+1

**Comment by matteberna** [comment]
+1

**Comment by wethinkall** [comment]
+1

**Comment by DylanSale** [comment]
+1

**Comment by omgware** [comment]
+1

**Comment by julienvillegas** [comment]
+1

**Comment by tzvetkovg** [comment]
+1

**Comment by antonyhg** [comment]
+1

**Comment by rindin** [comment]
+1

**Comment by tianqiujie** [comment]
+1

**Comment by marchy** [comment]
Absolutely. Library bindings are pivotal for any modern mobile project (and quite a lot of them actually). Please post more info on this topic (and real-life examples preferably). Perhaps you could take existing libraries (Mixpanel / Crashlytics / Appboy etc.)

Cheers!

**Comment by tianqiujie** [comment]
OK， I think there is no way to write robopods by ourselves. because you have to download the source code of robovm. but the source code is not open source anymore.

**Comment by yx421626416** [comment]
+1  I  think so.

**Comment by Allsimon** [comment]
+1
